code/controllers/subsystem/strings.dm

```diff
@@ -0,0 +1,11 @@
+SUBSYSTEM_DEF(strings)
+	name = "Strings"
+	init_order = INIT_ORDER_STRINGS
+	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/strings/Initialize(timeofday)
+	for(var/i in GLOB.string_files)
+		GLOB.vars[i] = file2list(GLOB.string_files[i])
```

Comment:
`GLOB.vars` screams to me like a bad idea. An admin could edit `GLOB.string_files` and then proccall `SSstrings.Initialize()` to overwrite protected vars, maybe if not to anything useful, but at least to garbage